sdk/tables/Azure.Data.Tables/api/Azure.Data.Tables.netstandard2.0.cs

```diff
@@ -186,3 +191,143 @@ public partial class TableResponseProperties
         public string TableName { get { throw null; } }
     }
 }
+namespace Azure.Data.Tables.Sas
+{
+    public partial class AccountSasBuilder
```

Comment:
These type names conflict with storage, we might want to prefix them.

Reply:
We might also have a problem with customers trying to build SAS tokens that include tables and other storage services and not being able to?

cc @tg-msft

Reply:
RE creating other service sas tokens, that is a good discussion to have. It could be confusing either way, I think, since this library only addresses tables.

Reply:
I think I'm going to remove all the account level sas types for now, and re-address this later to focus these changes on the SAS pattern that @tg-msft mentioned offline.

Reply:
I could see potentially including it here with a prefix and I could also see telling folks to go use `Azure.Storage.Common` if you want to create an account-level SAS.  The Storage folks have been looking at adding a `Tables` enum value for the service type so I don't think it'd be a terrible story.

Reply:
The other option would be Tables taking a dependency on `Azure.Storage.Common` for features like account-level SAS and `SharedKeyCredential`.  It'd add nontrivial overhead in terms of coordination and also make the APIs a lot messier to pull in both `Azure.Storage.*` and `Azure.Data.*` for champion scenarios.

Reply:
I would lean toward directing folks to use `Azure.Storage.Common` to create an account-level SAS while making sure our APIs are structured in a way that we could add account-level SAS to Tables in the future if it became a blocker for customers _(but I really don't think it ever will)_.  This would be worth starting an email conversation about with @KrzysztofCwalina, @JeffreyRichter, @pakrym, and @AlexGhiondea to make sure there's broad alignment before you rip anything out.

Reply:
> I could see potentially including it here with a prefix and I could also see telling folks to go use `Azure.Storage.Common` if you want to create an account-level SAS. The Storage folks have been looking at adding a `Tables` enum value for the service type so I don't think it'd be a terrible story.

I think the latter may be the more future proof approach as the storage tables / cosmos tables story evolves.